examples/pytorch/IterativePruning/alexnet.py

```diff
@@ -0,0 +1,201 @@
+# pylint: disable=W0221
+# pylint: disable=W0201
+# pylint: disable=W0223
+# pylint: disable=arguments-differ
+# pylint: disable=abstract-method
+
+import argparse
+import torch
+import time
+import os
+import torchvision
+import torchvision.transforms as transforms
+import pytorch_lightning as pl
+from torch.utils.data.dataloader import DataLoader
+from torch.utils.data import random_split
+import torch.nn as nn
+import torch.nn.functional as F
+import mlflow.pytorch
+from pytorch_lightning.metrics.functional import accuracy
+
+
+class DataModule(pl.LightningDataModule):
```

Comment:
How long does it take this run this example?

Reply:
Can we simplify this example by using a smaller dataset & model?

Reply:
@harupy Thank you for the review comments. I will start working on this. Will revert back to you as soon as we get anything concrete.

Reply:
@ankan94 Thanks! Here's what we usually consider when adding a new example:

1. How long does it take to run the example? Longer it takes, harder to test / review / debug / fix / maintain. 
2. Does the example download external data? If yes, how much disk space does the data consume?

We'd usually prefer a small model like a 3-layer MLP and a small dataset like the iris dataset.